src/data/news/software_releases.yml

```diff
@@ -1,4 +1,16 @@
-- 
+-
+  software: decred
+  version: 1.8.1
+  Permalink: "https://github.com/decred/decred-binaries/releases/tag/v1.8.1"
+  Params:
+    sortDate: "2023-10-09 16:00:22"
```

Comment:
The UTC timestamp I see on GitHub is `2023-10-09 16:37:22`. Not a big difference though. It would only matter if multiple releases came out on that day.

Reply:
Exactly, I didn't spend any time on setting the timestamps because they don't matter. They arent displayed, they're only used for sorting. I doubt any reader is going to care either.